living_documentation_generator/generator.py

```diff
@@ -58,7 +58,21 @@ class LivingDocumentationGenerator:
     PROJECT_ROOT = os.path.dirname(os.path.abspath(__file__))
 
     ISSUE_PAGE_TEMPLATE_FILE = os.path.join(PROJECT_ROOT, os.pardir, "templates", "issue_detail_page_template.md")
-    INDEX_PAGE_TEMPLATE_FILE = os.path.join(PROJECT_ROOT, os.pardir, "templates", "_index_page_html_template.md")
+    INDEX_NO_STRUCT_TEMPLATE_FILE = os.path.join(
+        PROJECT_ROOT, os.pardir, "templates", "_index_no_struct_page_template.md"
+    )
+    INDEX_ROOT_LEVEL_TEMPLATE_FILE = os.path.join(
+        PROJECT_ROOT, os.pardir, "templates", "_index_root_level_page_template.md"
+    )
+    INDEX_ORG_LEVEL_TEMPLATE_FILE = os.path.join(
+        PROJECT_ROOT, os.pardir, "templates", "_index_org_level_page_template.md"
+    )
+    INDEX_REPO_LEVEL_TEMPLATE_FILE = os.path.join(
+        PROJECT_ROOT, os.pardir, "templates", "_index_repo_level_page_template.md"
+    )
+
+    def __init__(self):
```

Comment:
Remove this __init__ method.

Reply:
Nice catch, the code is edited in commit: [a02c546](https://github.com/AbsaOSS/living-doc-generator/pull/60/commits/a02c546faee44c4ef2b374be4566e477e74160b6).